src/Pages/VideoList.js

```diff
@@ -0,0 +1,33 @@
+import { useData, ActionTypes, useAuth } from "../Context";
+import { Video } from "../Components";
+import { useEffect } from "react";
+export const VideoList = () => {
+  const { state, dispatch } = useData();
+  const {
+    authState: { isLoggedIn },
+  } = useAuth();
+  useEffect(() => {
+    document.title = "Home | Learn Finance";
+  }, []);
+  return (
+    <div>
+      <ul className="video-list">
+        {state.videos.map((item) => (
+          <div
+            key={item.id}
+            onClick={() => {
+              if (isLoggedIn) {
+                dispatch({
+                  type: ActionTypes.ADD_TO_HISTORY,
+                  payload: item,
+                });
+              }
+            }}
```

Comment:
OnClick handler can be moved to a separate function rather than inline handling. Tanay once said not to use inline onClick handler. Also, try to use the ternary operator rather than the old if-else conditions.